### ISSUE TITLE
Build failure: cbindgen fork no longer exists

### ISSUE THREAD

**Issue by lucidmachine** [nbigaouette/spell-rs#3]
The project depends upon [a fork of cbindgen](https://github.com/eqrion/cbindgen/pull/271), apparently to work around [a bug which has since been resolved](https://github.com/eqrion/cbindgen/issues/203). The specific revision referenced no longer exists.

```
$ cargo build
warning: unused manifest key: package.edition
warning: unused manifest key: package.edition
    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating git repository `https://github.com/Mrmaxmeier/cbindgen.git`        
error: failed to load source for a dependency on `cbindgen`                     

Caused by:
  Unable to update https://github.com/Mrmaxmeier/cbindgen.git?rev=996fa88ef7672042546b3ccd0e74453
9a3a645d7                                                                                       
```

**Comment by nbigaouette** [comment]
PR #4 was merged, and I've uploaded [`0.1.6`](https://github.com/nbigaouette/spell-rs/releases/tag/v0.1.6) to [crates.io](https://crates.io/crates/spell).

Thanks!

**Comment by lucidmachine** [comment]
You're welcome! Thanks for the quick merge and great project!
